src/GraphQL.Harness/Startup.cs

```diff
@@ -28,6 +30,11 @@ public void ConfigureServices(IServiceCollection services)
             // add execution components
             services.AddSingleton<IDocumentExecuter, DocumentExecuter>();
             services.AddSingleton<IDocumentWriter, DocumentWriter>();
+            services.AddSingleton<IErrorParser>(services =>
+            {
+                var settings = services.GetRequiredService<IOptions<GraphQLSettings>>();
+                return new ErrorParser(options => options.ExposeExceptions = settings.Value.ExposeExceptions);
+            });
```

Comment:
This is a sample of the code necessary to be added to a project that allows configuration of the `ExposeExceptions` property from within a settings object.  Simpler implementations could use something like this:
```csharp
services.AddSingleton<IErrorParser>(_ => new ErrorParser(o => o.ExposeExceptions = true));
```